packages/core/src/index.tsx

```diff
@@ -1,6 +1,9 @@
-export * from "./nodes";
-export * from "./render";
-export * from "./interfaces";
-export * from "./hooks";
 export * from "./editor";
 export * from "./events";
+export * from "./hooks";
+export * from "./interfaces";
+export * from "./nodes";
+export * from "./render";
+
+// We expose this methods for candu's internal usage. Use at your own risksi
+export { useInternalEditor } from "./editor/useInternalEditor";
```

Comment:
Is there a reason for needing this (I'm assuming its to access a certain internal action)? Otherwise, let's remove it.

Reply:
ah good catch let me remove it.